src/test/java/com/wikia/webdriver/testcases/mediatests/providers/PlayingVideoTests.java

```diff
@@ -76,11 +72,10 @@ public void PlayingVideoTests_004_ign() {
     video.verifyVideoIgnEmbed();
   }
 
-  @Test(groups = {"Media", "ProviderTests", "PlayingVideoTests", "PlayingVideoTests_005"})
-  @RelatedIssue(issueID = "SUS-64", comment = "Make sure that anyclip video "
-                                                 + "autoplays in the lightbox")
-  @Execute(disableFlash = "false", onWikia = "sktest123")
-  @InBrowser(browserSize = BROWSER_SIZE)
+  //TODO: Test disabled as Selenium doesn't allow to access elements with <object> tag. Check after Selenium update.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1135)